src/main/java/org/itsallcode/openfasttrace/maven/TraceMojo.java

```diff
@@ -94,6 +102,11 @@ public TraceMojo()
     @Override
     public void execute() throws MojoFailureException
     {
+        if (skip)
+        {
+            getLog().warn("Skipping OFT tracing");
```

Comment:
```suggestion
            getLog().warn("Skipping OFT tracing because property 'openfasttrace.skip' was set to 'true'.");
```
When skipping potentially important steps, please let users know why.